test/integration/update_lock_int_test.go

```diff
@@ -38,6 +38,10 @@ func (suite *UpdateIntegrationTestSuite) TestLocked() {
 }
 
 func (suite *UpdateIntegrationTestSuite) TestLockedChannel() {
+	targetBranch := "release"
+	if constants.BranchName == "release" {
+		targetBranch = "master"
+	}
```

Comment:
This is just temporary, because forwarding to the latest master branch does not work anymore, as we have multi-file updates there now.  These tests will have to be adapted soon anyways...